scripts/tests.e2e.sh

```diff
@@ -34,7 +34,7 @@ ACK_GINKGO_RC=true ginkgo build ./tests/e2e
 if [[ -n "${E2E_USE_PERSISTENT_NETWORK:-}" && -n "${TESTNETCTL_NETWORK_DIR:-}" ]]; then
   E2E_ARGS="--use-persistent-network"
 else
-  CAMINOGO_BIN_PATH="$(realpath ${CAMINOGO_BIN_PATH:-./build/caminogo})"
+  CAMINOGO_BIN_PATH="$(realpath "${CAMINOGO_BIN_PATH:-./build/caminogo}")"
   E2E_ARGS="--avalanchego-path=${CAMINOGO_BIN_PATH}"
```

Comment:
it's better to use --caminogo-path and change it in other files too

tests/e2e/e2e_test.go
tests/fixture/testnet/cmd/main.go
tests/fixture/testnet/local/README.md

Reply:
thats how it is in current dev
this PR is not about optimizing/refactoring, but about rebasing after-cortina changes on cortina-15
so, lets care about this with additional PR after dev branch is decided